crates/iota-core/src/congestion_tracker.rs

```diff
@@ -42,8 +39,15 @@ struct CongestionInfo {
 
     /// Lowest gas price of clearing transaction(s) accessing the object.
     lowest_clearing_gas_price: Option<u64>,
+
+    /// The hotness of an object corresponds to the expected tip to pay for a
+    /// successful execution.
+    pub hotness: f64,
```

Comment:
```suggestion
    hotness: f64,
```
Used only in this module, so does not need to be public.